src/solvers/solver_hardness.cpp

```diff
@@ -26,6 +26,8 @@ operator+=(const solver_hardnesst::sat_hardnesst &other)
   clauses += other.clauses;
   literals += other.literals;
   variables.insert(other.variables.begin(), other.variables.end());
+  clause_set.insert(
```

Comment:
Should we sort this?

Reply:
I was worried about too many calls to sort().